tribe-customizations/v2/week/grid-body/events-day/event.php

```diff
@@ -0,0 +1,141 @@
+<?php
+/**
+ * View: Week View - Event
+ *
+ * This is a template override of the file at:
+ * events-calendar-pro/src/views/v2/week/grid-body/events-day/event.php
+ *
+ * This template override is needed to make the The Events Calendar Pro Extension: Limit Week View Time Range
+ * work with the updated (V2) designs.
+ *
+ * See more documentation about our views templating system.
+ *
+ * @link    http://m.tri.be/1aiy
+ *
+ * @version 5.0.0
+ *
+ * @var WP_Post $event       The event post object with properties added by the `tribe_get_event` function.
+ * @var array   $ext_options An array of the extension settings.
+ *
+ * @see     tribe_get_event() For the format of the event object.
+ */
+
+use Tribe__Date_Utils as Dates;
+
+$classes = [ 'tribe-events-pro-week-grid__event' ];
+
+if ( ! empty( $event->is_past ) ) {
+	$classes[] = 'tribe-events-pro-week-grid__event--past';
+}
+
+if ( ! empty( $event->featured ) ) {
+	$classes[] = 'tribe-events-pro-week-grid__event--featured';
+}
+
+/*
+ * Some CSS classes (i.e. vertical position, duration and sequence) have been calculated in the Week View.
+ * Here we add them to the ones that should be applied to the event element.
+ */
+$classes = array_merge( array_values( $classes ), array_values( $event->classes ) );
+$classes = get_post_class( $classes, $event->ID );
+$data_js = [ 'tribe-events-pro-week-grid-event-link', 'tribe-events-tooltip' ];
+
+/**
+ * Get start time in seconds
+ */
+$start_time = Dates::time_between( $event->dates->start->format( 'Y-m-d 0:0:0' ), $event->dates->start->format( Dates::DBDATETIMEFORMAT ) );
+
+/**
+ * Get the extension settings
+ */
+$grid_start_time = (int) $ext_options['grid_start_time'];
+$grid_end_time   = (int) $ext_options['grid_end_time'];
+
+/**
+ * int containing if we have found the start time of an event based on the CSS class.
```

Comment:
```suggestion
 * Boolean containing if we have found the start time of an event based on the CSS class.
```
It's a sentence - it should start with a capital letter. Also, it's a boolean, not an int (true/false, not 1/0)

Reply:
You are absolutely right. What was I thinking! "Int"... sheeesh. :)

Thanks! Committed yours.